src/assertions/OrderAssertions.ts

```diff
@@ -158,11 +156,23 @@ export class OrderAssertions {
     orders: (TakerWalletOrder | ZeroExSignedFillOrder)[],
   ) {
     let makerTokensUsed = SetProtocolUtils.CONSTANTS.ZERO;
+    const requiredComponentFills: { [addr: string]: BigNumber } = {};
 
+    const {
+      makerTokenAmount,
+      quantity,
+    } = signedIssuanceOrder;
+
+    // Validate each of the orders and calculate fill amounts
     await Promise.all(
       _.map(orders, async (order: any) => {
+        // Count up components of issuance order that have been filled from this liquidity order
```

Comment:
We can rename `accumulateComponentAmounts` to something better so we make this comment obsolete.

Maybe `addLiquidityOrderContribution`...?